cmd/simulateBed/simulateBed.go

```diff
@@ -6,9 +6,20 @@ import (
 	"github.com/vertgenlab/gonomics/bed"
 	"github.com/vertgenlab/gonomics/common"
 	"log"
+	"math/rand"
+	"time"
 )
 
-func simulateBed(regionCount int, simLength int64, noGapFile string, outFile string) {
+func simulateBed(regionCount int, simLength int64, noGapFile string, outFile string, randSeed bool, setSeed int64) {
```

Comment:
Not a problem for now, but in the long-term this should probably be a separate pull request.